spec.go

```diff
@@ -244,6 +249,9 @@ func createLibcontainerConfig(cgroupName string, spec *specs.LinuxSpec) (*config
 		Capabilities: spec.Linux.Capabilities,
 		Readonlyfs:   spec.Root.Readonly,
 		Hostname:     spec.Hostname,
+		Labels: []string{
+			"bundle=" + cwd,
+		},
```

Comment:
Bundle has it's own property in the externalized json. He's only putting it in metadata in this internal list.